tests/integration/framework/process/placement/placement.go

```diff
@@ -61,15 +62,16 @@ func New(t *testing.T, fopts ...Option) *Placement {
 	uid, err := uuid.NewUUID()
 	require.NoError(t, err)
 
-	fp := util.ReservePorts(t, 4)
+	fp := ports.Reserve(t, 4)
+	intIP := fp.Port(t)
```

Comment:
```suggestion
	port := fp.Port(t)
```
Can this just be `port` for consistency sake, you have port further down for the slowappstart example. intIP isn't as self explanatory and its only used twice

Reply:
:+1: